internal/oci/uvm.go

```diff
@@ -309,10 +309,6 @@ func UpdateSpecFromOptions(s specs.Spec, opts *runhcsopts.Options) specs.Spec {
 		s.Annotations[annotations.MemorySizeInMB] = strconv.FormatInt(int64(opts.VmMemorySizeInMb), 10)
 	}
 
-	if _, ok := s.Annotations[annotations.GPUVHDPath]; !ok && opts.GPUVHDPath != "" {
```

Comment:
can you add a `// Deprecated: GPU VHDs are no longer supported` comment to `annotations.GPUVHDPath`?

Reply:
Do you want me to also add a deprecated tag on the runhcs options proto field? I'm nervous about doing that and causing compile warnings for people since we've never deprecated anything from that.

Reply:
lets leave that alone for now, yeah
itll probably cause issues for containerd too

Reply:
Sounds good. I added the deprecated comment on the annotation.